### PR TITLE
Adjusted several traps

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Adjusted bear traps and child objects.
Nerfed excelsior mines.
Added mines to maint spawn pools

## Why It's Good For The Game

Bear traps have more bite now, making them more than just a nuisance.
Low-Yield landmines now make traversing maint even more dangerous for the unwary. Eyes open!

## Changelog
```changelog
admin: moved the syzygy wire splice overrides to the occulus folder
balance: Excelsior mines are far less stupidly lethal now. They no longer throw fragments and deal a -1,-1,2,15 explosion down from -1, 1,2, 15. This will deal 70-90 damage to the target  depending on armor.
balance: bear traps and makeshift traps now deal the base damage instead of the fail damage when first stepped on. From 5 to 20 (16 for makeshift)
balance: bear traps can now be triggered by attacking them with any normal sized item.
add: landmines and emp landmines now have a chance to spawn as maint traps.
fix: excelsior landmines now no longer become un-deletable and infinitely exploding when picked up while active. Now they just explode
```

### CODE REVIEW

File: code/game/objects/items/weapons/traps.dm [hunk 1]
```diff
@@ -132,9 +132,18 @@ Freeing yourself is much harder than freeing someone else. Calling for help is a
 
 //Using a crowbar allows you to lever the trap open, better success rate
 /obj/item/weapon/beartrap/attackby(obj/item/C, mob/living/user)
```
Comment (zackerycodestoo) [inline code/game/objects/items/weapons/traps.dm:134]: Non-blocker -- would it be possible to add code to trigger traps by throwing things at them?

File: code/game/objects/items/weapons/traps.dm [hunk 2]
```diff
@@ -132,9 +132,18 @@ Freeing yourself is much harder than freeing someone else. Calling for help is a
...
-	if (C.has_quality(QUALITY_PRYING))
+	if (C.has_quality(QUALITY_PRYING) && buckled_mob)
 		attempt_release(user, C)
 		return
+	if (deployed && C.w_class > ITEM_SIZE_SMALL)//Occulus Edit Start - Triggering traps with medium items!
```
Comment (zackerycodestoo) [inline code/game/objects/items/weapons/traps.dm:138]: Non-blocker -- this code also allows things like triggering traps with labcoats, etc. with no ill effects. Maybe an item damage component?

**Review by zackerycodestoo — APPROVED** [review]
Non-blocking commentary -- approved

File: code/game/objects/items/weapons/traps.dm (hunk 2 above)
Comment (Jamini) [reply code/game/objects/items/weapons/traps.dm:138]: Eh, anything heavy enough should be able to trip it. I would include a check that forces the item out of your hand, but that might cause issues with cyborg and implants.

**Pull request merged**